ksp_plugin/planetarium.cpp

```diff
@@ -236,6 +244,12 @@ RP2Lines<Length, Camera> Planetarium::PlotMethod2(
       continue;
     }
 
+    if (minimal_distance != nullptr) {
```

Comment:
Why not do that all the time?  It's not overly costly and it would make the code a bit simpler.

Reply:
That sounds like trading a well-predicted branch for a poorly-predicted one; if we weren’t trying to optimize the square roots by working with squared distances, we would need a null check anyway.